src/main/java/taxiservice/web/filters/AuthenticationFilter.java

```diff
@@ -0,0 +1,57 @@
+package taxiservice.web.filters;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxiservice.Application;
+import taxiservice.lib.Injector;
+import taxiservice.service.DriverService;
+
+public class AuthenticationFilter implements Filter {
+    private static final String DRIVER_ID = "driver_id";
+    private static final Injector injector =
+            Injector.getInstance(Application.class.getPackageName());
+    private static final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+    private static final Set<String> ALLOWED_URL = new HashSet<>();
```

Comment:
```suggestion
    private static final Set<String> allowedUrls = new HashSet<>();
```
it's not a constant (not deeply immutable, the same situation as with injector)